app/answerOperations.js

```diff
@@ -0,0 +1,58 @@
+var Question = require('../models/question');
```

Comment:
Lets use strict mode for all the ``.js`` files. 
Here is the docs for it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode